packages/connector-aliyun-sms/package.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "@logto/connector-aliyun-sms",
+    "version": "0.1.0",
+    "description": "Aliyun SMS connector implementation.",
+    "main": "./lib/index.js",
+    "exports": "./lib/index.js",
+    "author": "Silverhand Inc. <contact@silverhand.io>",
+    "license": "MPL-2.0",
+    "files": [
+        "lib",
+        "docs"
+    ],
+    "private": true,
```

Comment:
false

Reply:
and also need to aligen for other packages.

Reply:
fixed and others also aligned.